cfme/common/__init__.py

```diff
@@ -261,72 +262,68 @@ class Taggable(object):
     standardized widgetastic views
     """
 
-    def add_tag(self, category=None, tag=None, cancel=False, reset=False, details=True):
+    def add_tag(self, tag=None, cancel=False, reset=False, details=True, entities=None):
         """ Add tag to tested item
 
         Args:
-            category: category(str)
-            tag: tag(str) or Tag object
+            tag: Tag object
             cancel: set True to cancel tag assigment
             reset: set True to reset already set up tag
             details (bool): set False if tag should be added for list selection,
                             default is details page
+            entities(list): list of entities or entities.names to tag
         """
-        if details:
+        if details and not entities:
             view = navigate_to(self, 'EditTagsFromDetails')
-        else:
-            view = navigate_to(self, 'EditTags')
-        if isinstance(tag, Tag):
-            category = tag.category.display_name
-            tag = tag.display_name
+        elif entities or not details:
+            view = navigate_to(self, 'EditTags', *entities)
```

Comment:
**QUESTION:** What happens here if entities is None?   The check above could allow us to get here if there are no details and entities is None if I understand correctly?   Is that what you really want?

Reply:
I've added entities as an option to tag multiple items, but we can do it only in list view so if the entity is None only one item will be tagged, the one that tagging function is called, the same if details is False, we tag via the list

Reply:
So, in the end, I decided to move this changes out of this PR, as this should be only for collections, and will be done in separate PR